crypto/evp/evp_lib.c

```diff
@@ -700,3 +701,130 @@ int EVP_hex2ctrl(int (*cb)(void *ctx, int cmd, void *buf, size_t buflen),
     OPENSSL_free(bin);
     return rv;
 }
+
+#ifndef FIPS_MODE
+/*
+ * TODO(3.0): Temporarily unavailable in FIPS mode. This will need to be added
+ * in later.
+ */
+
+#define MAX_PARAMS 10
+typedef struct {
+    /* Number of the current param */
+    size_t curr;
+    /* Key for the current param */
+    const char *key[MAX_PARAMS];
```

Comment:
Isnt it cleaner to put the key, bnparam, bufsz into a seperate struct.. and have an array of them?

Reply:
Done